src/components/CourtReservationDetails.js

```diff
@@ -43,117 +49,116 @@ class CourtReservationDetails extends Component {
+  render() {
+    const { court, startDate, endDate, light, commerce } = this.props;
 
-        return (
-            <View style={styles.mainContainer}>
-                <Avatar
-                    rounded
-                    source={profilePicture ? { uri: profilePicture } : null}
-                    size={90}
-                    icon={{ name: commerce ? 'store' : 'person' }}
-                    containerStyle={styles.avatarStyle}
-                />
-                <View style={styles.contentContainer}>
-                    <CardSection style={[styles.cardSections, { paddingBottom: 0 }]}>
-                        <Text style={styles.bigText}>
-                            {name}
-                        </Text>
-                    </CardSection>
-                    {/*this.renderAddress()*/}
-                    <CardSection style={[styles.cardSections, { paddingTop: 8, paddingBottom: 0 }]}>
-                        <Text style={styles.mediumText}>
-                            {court.name}
-                        </Text>
-                    </CardSection>
-                    <CardSection style={[styles.cardSections, { paddingBottom: 0 }]}>
-                        <Text style={styles.regularText}>
-                            {`${court.court} - ${court.ground}`}
-                        </Text>
-                    </CardSection>
-                    <CardSection style={styles.cardSections}>
-                        <Text style={styles.regularText}>
-                            {light ? 'Con Luz' : 'Sin Luz'}
-                        </Text>
-                    </CardSection>
-                    <Divider style={styles.divider} />
-                    <CardSection style={[styles.cardSections, { paddingBottom: 0 }]}>
-                        <Text style={styles.regularText}>
-                            {`${DAYS[moment(startDate).day()]} ${moment(startDate).format('D')} de ${MONTHS[moment(startDate).month()]}`}
-                        </Text>
-                    </CardSection>
-                    <CardSection style={styles.cardSections}>
-                        <Text style={styles.regularText}>
-                            {`De ${moment(startDate).format('HH:mm')} hs. a ${moment(endDate).format('HH:mm')} hs.`}
-                        </Text>
-                    </CardSection>
-                    {this.renderPrice()}
-                </View>
-            </View>
-        );
-    }
+    const { name, profilePicture } = this.state;
 
+    return (
+      <View style={styles.mainContainer}>
+        <Avatar
+          rounded
+          source={profilePicture ? { uri: profilePicture } : null}
+          size={90}
+          icon={{ name: commerce ? 'store' : 'person' }}
+          containerStyle={styles.avatarStyle}
+          onPress={() => {
+            if (this.props.client)
+              this.props.navigation.navigate('clientProfileView', {
+                clientId: this.props.client.id
+              });
+          }}
```

Comment:
Yo aca pondria `onPress={this.props.onProfilePicturePress}` cosa de que sea una `prop` y se la pasas cuando llames a este componente, asi se navega el perfil del cliente o del negocio dependiendo el caso, sin tener que hacer este `if` aca.